src/Equinor.ProCoSys.BusSender.Worker/Program.cs

```
@@ -143,9 +147,27 @@ private static IHostBuilder CreateHostBuilder(string[] args)
     public static async Task Main(string[] args)
     {
         using var host = CreateHostBuilder(args).Build();
-        ILogger? logger = host.Services.GetService<ILogger<Program>>();
+        using var scope = host.Services.CreateScope();
+        var plantService = host.Services.GetService<IPlantService>();
+        var plantRepository = scope.ServiceProvider.GetService<IPlantRepository>();
+        var plants = new List<string>();
+
+        if (plantRepository != null)
+        {
+            plants = await plantRepository.GetAllPlantsAsync();
+        }
+
+        plantService?.RegisterPlantsHandledByCurrentInstance(host, plants);

```

Similarly, if plantService is null, RegisterPlantsHandledByCurrentInstance will not be run, what are the consequences of this?

---

Implementing fail fast for this.